test/run_as_user/run_as_user_test.go

```diff
@@ -40,15 +40,23 @@ func TestRunAsUser(t *testing.T) {
 
 	for _, parameter := range parameters {
 		t.Run(fmt.Sprintf("resource file location %s user %s", parameter.dataInput, parameter.user), func(t *testing.T) {
-			test.CopyFile(parameter.dataInput, configOutputPath)
-			test.StartAgent(configOutputPath, true)
+			common.CopyFile(parameter.dataInput, configOutputPath)
+			common.StartAgent(configOutputPath, true)
 			time.Sleep(agentRuntime)
 			log.Printf("Agent has been running for : %s", agentRuntime.String())
 			// Must read the pid file while agent is running
-			pidOutput := test.RunCommand(test.CatCommand + pidFile)
-			agentOwnerOutput := test.RunCommand(test.AppOwnerCommand + pidOutput)
+			pidOutput, err := common.RunCommand(common.CatCommand + pidFile)
+			if err != nil {
+				t.Fatalf("Error: %v", err)
```

Comment:
Why `Fatal` here but `Error` below?

Reply:
To be honest, I'm not sure. The `RunCommand` previously return Fatal so I preserve the order. However, in the end, `Fatalf` and `Errorf` will produce the same results but Fatal with fail earlier so its better to use Fatal here